developer-docs-site/docs/concepts/delegated_staking.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Delegated Staking"
+slug: " Delegated staking"
+---
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Delegated Staking
+
+## Delegated Staking on the Aptos blockchain
+
+:::tip Consensus We strongly recommend that you read the consensus section of Aptos Blockchain Deep Dive before proceeding further. :::
```

Comment:
since this is an extension of staking protocol, feel this should replace by asking people to read the staking doc first